README.md

```diff
@@ -22,19 +22,21 @@ The clr.fund smart contracts consist of a factory contract that deploys a new co
 The clr.fund application can use any [EVM-compatible chain](https://ethereum.org/) as a backend. The application can be hosted on [IPFS](https://ipfs.io/) and can also run locally.
 
 For more details, see the [sequence diagram](docs/clrfund.svg) and [clr.fund constitution](https://github.com/clrfund/constitution).
-![sequence diagram](docs/clrfund.svg)
```

Comment:
What do you think about removing this image from the readme? I think that the image is too big to be there and you have the link to access it in the previous line.

Reply:
Agreed.